### DISCUSSION TITLE
Is it possible to share RSDs?

### DISCUSSION THREAD

**Discussion by Ekmiec** [wgu-opensource/osmt#90] [Asking for Help] [answered]
I was under the impression you can share RSDs from repositories. Is this feature available and if so, is there a “How To” to get started?

**Comment by JohnKallies** [comment]
@arblackwell1 when you get back, can you please answer this?

**Comment by arblackwell1** [comment, marked as answer]
@Ekmiec The way to share RSDs is indicated in the How-To here: [Export a Single RSD or an RSD Collection](https://osmt.io/docs.html)
